scripts/run.py

```diff
@@ -0,0 +1,224 @@
+        config = yaml.safe_load(file)
+    # read config.default.yaml file
+    with open("config/config.default.yaml", 'r') as file:
+        config_default = yaml.safe_load(file)
+    # scenario params
+    params = config["scenario"]
+    params_default = config_default["scenario"]
+    ll, clusters, sector_opts, planning_horizons = params['ll'][0], params['clusters'][0], params['sector_opts'][0], params['planning_horizons'][0]
+    simpl, opts = params_default['simpl'][0], params_default['opts'][0]
+    filename = f"elec_s{simpl}_{clusters}_l{ll}_{opts}_{sector_opts}_{planning_horizons}.nc"
+    return filename
+
+
+def increase_biomass_potential(factor=1.2):
+    # change path to pypsa-eur
+    change_path_to_pypsa_eur()
+
+    # Define the file path
+    file_path = 'scripts/prepare_sector_network.py'
+
+    # Define the line to be added
+    new_line = f'    biomass_potentials = {factor} * biomass_potentials\n'
+
+    # Read the contents of the file
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+
+    # Find the index of the line containing the specified text
+    index = next((i for i, line in enumerate(lines) if 'biomass_potentials = pd.read_csv(snakemake.input.biomass_potentials, index_col=0)' in line), None)
+
+    # Insert the new line after the specified line
+    if index is not None:
+        lines.insert(index + 1, new_line)
+
+    # Write the modified contents back to the file
+    with open(file_path, 'w') as file:
+        file.writelines(lines)
+
+    # log changes
+    logging.info(f"Increase biomass potentials by {factor} factor")
+
+    # move to base directory
+    change_path_to_base()
+
+
+def revert_biomass_potential():
+    # Change path to pypsa-eur
+    change_path_to_pypsa_eur()
+
+    # Define the file path
+    file_path = 'scripts/prepare_sector_network.py'
+
+    # Read the contents of the file
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+
+    # Find the index of the line containing the specified text
+    index = next((i for i, line in enumerate(lines) if 'biomass_potentials = 1.2 * biomass_potentials' in line), None)
+
+    # Remove the line if found
+    if index is not None:
+        del lines[index]
+
+    # Write the modified contents back to the file
+    with open(file_path, 'w') as file:
+        file.writelines(lines)
+
+    # log changes
+    logging.info(f"Revert biomass potentials back")
+
+    # Move to base directory
+    change_path_to_base()
+
+
+def prepare_prenetwork(scenario, horizon):
+    # get config path
+    config_path = get_config_path(scenario, horizon)
+    # config path relative to pypsa-eur folder
+    config_path = "../../" + config_path
+
+    # change path to pypsa-eur
+    change_path_to_pypsa_eur()
+
+    # get .nc filename
+    filename = get_network_name(scenario, horizon)
+    # run prenetwork
+    command = f"snakemake -call results/{scenario}/prenetworks/{filename} --configfile {config_path} --force"
```

Comment:
```suggestion
    command = f"snakemake -call prepare_sector_networks --configfile {config_path} --force"
```
similar to `solve_sector_network`, there is a `preapre_sector_network` rule
why is the flag `--force` needed?

Reply:
Of course using prepare_sector_networks is easiest and the first thing that come to my mind. And I tried it. But if has some major flaw. It does not sense that the network was updated (I will clarify in the end). I have tested it. So it should be filename. It ensures 100% that prepare_sector_networks will run. If you accidentally manually run `set_capacities` rule. Then run `snakemake -call prepare_sector_networks --force` then nothing will be run. It does not sense that file was changed. But if you run `snakemake -call results/{scenario}/prenetworks/{filename} --force ...` than it will 100% run.